manage/installing/installation_windows10_wsl.rst

```diff
@@ -0,0 +1,271 @@
+================================================================
+Installing Plone on a Windows 10 Linux Subsystem for Development
+================================================================
+
+A manual for installing a `Plone CLI <https://pypi.org/project/plonecli/>`_ (and Python 3) based development environment into a Windows Linux Subsystem (WSL) under Windows 10.
+Ubuntu is used as the Linux distribution.
+It is possible to edit the addons created by Plone CLI with an IDE of your choice under Windows.
+
+Introduction
+============
+
+The following tutorial will lead you trought all the steps to install and use Plone CLI on your Windows device.
+
+Ubuntu
+======
+
+Pre Settings
+------------
+
+Before installing the shell to later configure in, the first thing to do is to enable the optional feature for a subsystem on windows.
+Therefor, the programm Windows PowerShell has to be launched (as administrator! - right-click the application and choose "launch as admin").
+Do not launch the Powershell ISE application, because it won't work.
+In the shell the following command has to be entered.
+
+.. code-block :: powershell
```

Comment:
I'm not sure this Pygments lexer works. I did not have much luck in Pyramid docs. If it does not, then [choose another](http://pygments.org/docs/lexers/#lexers-for-various-shells).